index.js

```diff
@@ -1,15 +1,69 @@
 class SortedList {
-  constructor() {}
+  constructor(items, length) {
+    this.items = [];
+    this.length = 0;
+  }
 
-  add(item) {}
+  add(item) {
+    let newArr = this.items;
+    newArr.push(item)
+    newArr.sort((a, b)=>{return a-b})
+     this.length++
+  
+  }
 
-  get(pos) {}
+  get(pos) {
+    if(pos <= this.length-1){
```

Comment:
Podríamos refactorizar si cumple la condicion es un indice de los elementos entonces no es necesario iterar el arreglo podemos acceder al elemento con el mismo pos
```suggestion
    if(pos <= this.length-1){
    return this.items[pos]}
```